docs/scipp-neutron/from-mantid-to-scipp.ipynb

```diff
@@ -308,7 +312,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data = sc.concatenate(data1, data2, 'spectrum')"
+    "data = sc.concatenate(da, da, 'spectrum')"
```

Comment:
Using same input twice may be confusing?

Reply:
I agree that 2 different inputs would be better. I think it's standard for developers who understand that you often do this when you are testing stuff (e.g. in unit tests) but users might be confused and think as to what is trying to be achieved here, and some may wonder if it only works with twice the same input ?

Reply:
Completely agree with the statement above. I'll change to something meaningful.